compiler/luci/pass/src/QuantizeDequantizeWeightsPass.cpp

```diff
@@ -322,7 +322,7 @@ struct QuantizeDequantizeWeights final : public luci::CircleNodeMutableVisitor<v
     }
     else
     {
-      sym_wquant_per_channel(weights, min, max, scaling_factor, zp, nudged_min, nudged_max);
+      sym_wquant_per_channel(weights, min, max, scaling_factor, nudged_min, nudged_max);
```

Comment:
`zp` is constructed with `std::vector<int64_t> zp(min.size());`.
 
It may be better to update to `std::vector<int64_t> zp(min.size(), 0);` for clarity ?

Reply:
As I understand, constructor with size will initialize the member with default value. I don't think the 2nd parmeter is necessary. I personally this form is succinct and enough. But if you prefer, I will add it.